common/core/build.gradle.kts

```diff
@@ -77,9 +64,11 @@ kotlin {
 
 android {
     compileSdk = Versions.androidCompileSdk
+    buildToolsVersion = Versions.androidBuildToolsVersion
     sourceSets["main"].manifest.srcFile("src/androidMain/AndroidManifest.xml")
     defaultConfig {
         minSdk = Versions.androidMinSdk
+        targetSdk = Versions.androidTargetSdk
```

Comment:
I believe `targetSdk` is not meaningful for library modules as the application module's `targetSdk` is used. Do you have information indicating otherwise?

Reply:
I thing you are right. I usually set all 4 properties: `compileSdk`, `buildToolsVersion`, `minSdk`, `targetSdk`.